repos/system_upgrade/el7toel8/actors/checkbtrfs/actor.py

```diff
@@ -22,14 +22,21 @@ def process(self):
         for fact in self.consume(ActiveKernelModulesFacts):
             for active_module in fact.kernel_modules:
                 if active_module.filename == 'btrfs':
-                    report_with_remediation(
-                        title='Btrfs removed in the next major version',
-                        summary='The Btrfs file system was introduced as Technology Preview with the initial release '
-                                'of Red Hat Enterprise Linux 6 and Red Hat Enterprise Linux 7. As of versions 6.6 and '
-                                '7.4 this technology has been deprecated and will be removed in next major version.',
-                        remediation='Please consider migrating your Btrfs mount point(s) to a different filesystem '
-                                    'before next upgrade attempt. If no Btrfs filesystem is in use, please unload '
-                                    'btrfs kernel module running "# rmmod btrfs".',
-                        severity='high',
-                        flags=['inhibitor'])
+                    create_report(
+                        reporting.Title('Btrfs has been removed from RHEL8'),
+                        reporting.Summary('The Btrfs file system was introduced as Technology Preview with the initial
+                            'release of Red Hat Enterprise Linux 6 and Red Hat Enterprise Linux 7. As of versions 6.6 '
+                            'and 7.4 this technology has been deprecated and removed in RHEL8.'),
+                        reporting.ExternalLink(
+                            title='Considerations in adopting RHEL 8 - btrfs has been removed.',
+                            url='https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/considerations_in_adopting_rhel_8/file-systems-and-storage_considerations-in-adopting-rhel-8#btrfs-has-been-removed_file-systems-and-storage'
+                        )
+                        reporting.Severity('high'),
```

Comment:
```suggestion
                        reporting.Severity(reporting.Severities.HIGH),
```
Any chance to add this?

Reply:
BTW, very nice work congrats.

Reply:
Original idea is @shaded-enmity 's :)

Reply:
was thinking about the "severity stuff" too...

Reply:
severities implemented. Thanks for the suggestions.